devicetypes/qubino/qubino-dimmer.src/qubino-dimmer.groovy

```diff
@@ -226,12 +226,9 @@ def configure() {
 		Group 5: Multilevel sensor report (external temperature sensor report).
 
 	*/
-	commands << zwave.associationV1.associationSet(groupingIdentifier:1, nodeId:[zwaveHubNodeId])
-	commands << zwave.associationV1.associationSet(groupingIdentifier:2, nodeId:[zwaveHubNodeId])
-	commands << zwave.associationV1.associationSet(groupingIdentifier:3, nodeId:[zwaveHubNodeId])
-	commands << zwave.associationV1.associationSet(groupingIdentifier:4, nodeId:[zwaveHubNodeId])
-	commands << zwave.associationV1.associationSet(groupingIdentifier:5, nodeId:[zwaveHubNodeId])
-	commands << zwave.associationV1.associationSet(groupingIdentifier:6, nodeId:[zwaveHubNodeId])
+	commands << zwave.multiChannelAssociationV2.multiChannelAssociationRemove(groupingIdentifier:1, nodeId:[])
+	commands << zwave.multiChannelAssociationV2.multiChannelAssociationSet(groupingIdentifier:1, nodeId:[zwaveHubNodeId])
+	commands << zwave.multiChannelAssociationV2.multiChannelAssociationGet(groupingIdentifier: 1)
```

Comment:
Is this `get` being handled anywhere? If not you should probably just remove it.

Reply:
What did the mutlichannel association table look like before you sent the remove?

Reply:
@greens Can You confirm that this is configured by the hub already?

Reply:
I double-checked how the device works without this configuration - qubino device didn't report anything. 
If the hub somehow configures it itself before a DTH comes to play - whatever it does, it has to be overwritten by this DTH configuration.